### PR TITLE
minor fixes for docs and code

### PR DESCRIPTION
**Pull request by tripledes**
* Minor fixes on code and documentation
* Add last additions to the doctrees
* Update HTML docs

### CODE REVIEW

File: thesheriff/application/outlaw/list_gangs.py [hunk 1]
```diff
@@ -5,7 +5,7 @@
 
 
 class ListGangs:
-    """Class ListGangs implements the list gangs use case.
+    """Class ListGangs implements the List Outlaw's Gangs use case.
```
Comment (rmuhamed) [inline thesheriff/application/outlaw/list_gangs.py]: 👍

File: thesheriff/application/raid/end_raid.py [hunk 2]
```diff
@@ -11,13 +11,13 @@ class EndRaid:
     def __init__(self, raid_repository: RaidRepository):
         self.raid_repository = raid_repository
 
-    def execute(self, raid_id: int, raid_score: int):
-        """execute is the actual action of the Raid rating use case.
+    def execute(self, raid_id: int, raid_score: float):
```
Comment (rmuhamed) [inline thesheriff/application/raid/end_raid.py]: Maybe a `EndRaidRequest `class holding those two values? Just in order to have same "expected" object at execute method.

Worth it?

File: thesheriff/domain/outlaw/repository/outlaw_repository.py [hunk 3]
```diff
@@ -13,7 +13,7 @@ def of_id(self, outlaw_id: int) -> Outlaw:
         pass
 
     @abc.abstractmethod
-    def add(self, new_outlaw: Outlaw) -> int:
+    def add(self, new_outlaw: Outlaw) -> Outlaw:
```
Comment (rmuhamed) [inline thesheriff/domain/outlaw/repository/outlaw_repository.py]: I prefer to return the entire object with the id, make more sense to me (and I guess is the usual way) 🥇

File: thesheriff/domain/raid/repository/raid_repository.py [hunk 4]
```diff
@@ -13,7 +13,7 @@ def of_id(self, raid_id: int) -> Raid:
         pass
 
     @abc.abstractmethod
-    def add(self, new_raid: Raid) -> NoReturn:
+    def add(self, new_raid: Raid) -> Raid:
```
Comment (rmuhamed) [inline thesheriff/domain/raid/repository/raid_repository.py]: 🥇

File: thesheriff/infrastructure/repository/mysql_gang_repository.py [hunk 5]
```diff
@@ -64,21 +64,21 @@ def all(self) -> List[Gang]:
             gangs.append(gang)
         return gangs
 
-    def add(self, new_gang: Gang) -> NoReturn:
+    def add(self, new_gang: Gang) -> Gang:
```
Comment (rmuhamed) [inline thesheriff/infrastructure/repository/mysql_gang_repository.py]: 🥇

File: thesheriff/application/raid/end_raid.py (hunk 2 above)
Comment (tripledes) [reply thesheriff/application/raid/end_raid.py]: makes sense! I'm adding it and will update the PR.

**Pull request merged**